### PR TITLE
Update Senarai_Order.txt

### PR DESCRIPTION
**Pull request by ibnusulaiman83**

### CODE REVIEW

File: Senarai_Order.txt [hunk 1]
```diff
@@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
 21, Akirien, 1, M 1
 22, Lina, 1, L, 1
 23, AmirolAhmad, 1, M 1
+24, Fairuz, 1, L, 1 (via PM FB)
```
Comment (ApOgEE) [inline Senarai_Order.txt:30]: nak tanya bro, Fairuz ni PM kat bro ke?

Comment (ibnusulaiman83) [inline Senarai_Order.txt:30]: yes bos

On Thu, Jun 25, 2015 at 7:52 PM, M. Fauzilkamil Zainuddin <
notifications@github.com> wrote:

> In Senarai_Order.txt
> https://github.com/ApOgEE/TempahTeeGit/pull/45#discussion_r33246402:
> 
> > @@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
> >  21, Akirien, 1, M 1
> >  22, Lina, 1, L, 1
> >  23, AmirolAhmad, 1, M 1
> > +24, Fairuz, 1, L, 1 (via PM FB)
> 
> nak tanya bro, Fairuz ni PM kat bro ke?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ApOgEE/TempahTeeGit/pull/45/files#r33246402.

## 

Regards,

Fairuz Sulaiman

Web Developer, System Analyst
+6019-3042-711

https://wikiabout.me/fairuz.sulaiman
http://facebook.com/permata.biru83
http://twitter.com/ibnusulaiman

Comment (ApOgEE) [inline Senarai_Order.txt:30]: kalau dia PM kat bro, better betulkan entry tu. just tambah order dia sekali dalam order bro. supaya saya tak pening nanti nak cari FB dia mana. sebab belum tentu dia add saya lagi kat FB. So, tolong betulkan entry tu. buat satu entry sahaja. supaya saya boleh refer untuk payment nanti.

Comment (ibnusulaiman83) [inline Senarai_Order.txt:30]: Lagi 1
Kamarool , 1, S,1

2015-06-26 1:27 GMT+08:00 M. Fauzilkamil Zainuddin <notifications@github.com

> :
> 
> In Senarai_Order.txt
> https://github.com/ApOgEE/TempahTeeGit/pull/45#discussion_r33280987:
> 
> > @@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
> >  21, Akirien, 1, M 1
> >  22, Lina, 1, L, 1
> >  23, AmirolAhmad, 1, M 1
> > +24, Fairuz, 1, L, 1 (via PM FB)
> 
> kalau dia PM kat bro, better betulkan entry tu. just tambah order dia
> sekali dalam order bro. supaya saya tak pening nanti nak cari FB dia mana.
> sebab belum tentu dia add saya lagi kat FB. So, tolong betulkan entry tu.
> buat satu entry sahaja. supaya saya boleh refer untuk payment nanti.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ApOgEE/TempahTeeGit/pull/45/files#r33280987.

## 

Regards,

Fairuz Sulaiman

Web Developer, System Analyst
+6019-3042-711

https://wikiabout.me/fairuz.sulaiman
http://facebook.com/permata.biru83
http://twitter.com/ibnusulaiman

Comment (ApOgEE) [inline Senarai_Order.txt:30]: ye tuan, tolong jadikan satu line sahaja. contoh:
`24, Fairuz & Kamarool, 2, S 1, L 1`
supaya saya boleh refer order ni dari tuan sahaja. nanti tuan setel payment dia sekali je. tak perlu bayar asing2

Comment (ibnusulaiman83) [inline Senarai_Order.txt:30]: Ok baik tuan yang disayangi tuan. Bagi no akaun cimb /maybank

2015-06-26 13:27 GMT+08:00 M. Fauzilkamil Zainuddin <
notifications@github.com>:

> In Senarai_Order.txt
> https://github.com/ApOgEE/TempahTeeGit/pull/45#discussion_r33329497:
> 
> > @@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
> >  21, Akirien, 1, M 1
> >  22, Lina, 1, L, 1
> >  23, AmirolAhmad, 1, M 1
> > +24, Fairuz, 1, L, 1 (via PM FB)
> 
> ye tuan, tolong jadikan satu line sahaja. contoh:
> 24, Fairuz & Kamarool, 2, S 1, L 1
> supaya saya boleh refer order ni dari tuan sahaja. nanti tuan setel
> payment dia sekali je. tak perlu bayar asing2
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ApOgEE/TempahTeeGit/pull/45/files#r33329497.

## 

Regards,

Fairuz Sulaiman

Web Developer, System Analyst
+6019-3042-711

https://wikiabout.me/fairuz.sulaiman
http://facebook.com/permata.biru83
http://twitter.com/ibnusulaiman

Comment (ApOgEE) [inline Senarai_Order.txt:30]: haha... dah update ke belum patch ni? payment saya akan kutip minggu depan.

Comment (ibnusulaiman83) [inline Senarai_Order.txt:30]: ok la tu. Kamarool punya macam mana?

2015-06-27 9:54 GMT+08:00 M. Fauzilkamil Zainuddin <notifications@github.com

> :
> 
> In Senarai_Order.txt
> https://github.com/ApOgEE/TempahTeeGit/pull/45#discussion_r33408136:
> 
> > @@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
> >  21, Akirien, 1, M 1
> >  22, Lina, 1, L, 1
> >  23, AmirolAhmad, 1, M 1
> > +24, Fairuz, 1, L, 1 (via PM FB)
> 
> haha... dah update ke belum patch ni? payment saya akan kutip minggu depan.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ApOgEE/TempahTeeGit/pull/45/files#r33408136.

## 

Regards,

Fairuz Sulaiman

Web Developer, System Analyst
+6019-3042-711

https://wikiabout.me/fairuz.sulaiman
http://facebook.com/permata.biru83
http://twitter.com/ibnusulaiman

Comment (ApOgEE) [inline Senarai_Order.txt:30]: Kamarool punya tu ada beberapa cara:
1. tambah order dia dalam order ko. nanti masa payment, ko bayar sekali
2. suruh dia buat pull request sendiri supaya aku boleh update payment detail kat dia
3. kalau dia tak reti guna git, ko boleh suruh dia add aku kat FB dan suruh dia PM aku untuk order dia

Jadi, sekarang ni ko kena betulkan entry ni, buat patch baru, dan hantar pull request

Comment (ibnusulaiman83) [inline Senarai_Order.txt:30]: Apa kata tuan @kamarool di Jomweb suruh di pull request sendiri.

2015-06-27 18:49 GMT+08:00 M. Fauzilkamil Zainuddin <
notifications@github.com>:

> In Senarai_Order.txt
> https://github.com/ApOgEE/TempahTeeGit/pull/45#discussion_r33411788:
> 
> > @@ -27,3 +27,5 @@ Masukkan order korang dalam bentuk CSV (Comma Separated Value)... hahaha...
> >  21, Akirien, 1, M 1
> >  22, Lina, 1, L, 1
> >  23, AmirolAhmad, 1, M 1
> > +24, Fairuz, 1, L, 1 (via PM FB)
> 
> Kamarool punya tu ada beberapa cara:
> 1. tambah order dia dalam order ko. nanti masa payment, ko bayar sekali
> 2. suruh dia buat pull request sendiri supaya aku boleh update payment
> detail kat dia
> 3. kalau dia tak reti guna git, ko boleh suruh dia add aku kat FB dan
> suruh dia PM aku untuk order dia
> 
> Jadi, sekarang ni ko kena betulkan entry ni, buat patch baru, dan hantar
> pull request
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ApOgEE/TempahTeeGit/pull/45/files#r33411788.

## 

Regards,

Fairuz Sulaiman

Web Developer, System Analyst
+6019-3042-711

https://wikiabout.me/fairuz.sulaiman
http://facebook.com/permata.biru83
http://twitter.com/ibnusulaiman

Comment (ApOgEE) [inline Senarai_Order.txt:30]: ok, suruh dia pull request sendiri... so, sekarang ni saya merge Fairuz punya sahaja lah ya?

**Comment by ApOgEE** [comment]
ok dah merge.... saya hantar borang bayaran. jangan ketinggalan untuk buat bayaran nanti. kalau tak bayar, tak dapat baju. terima kasih :+1:

**Comment by sothye** [comment]
Roger. Tqvm.

**Pull request merged**